test/syscall/wt2336_base/base.run

```diff
@@ -46,15 +46,15 @@ SYSTEM("Darwin");
 #define FTRUNCATE(fd, len)			/* do nothing */
 #endif
 
-TRACE("close,fdatasync,fsync,ftruncate,open,pwrite,rename");
+TRACE("close,fdatasync,fsync,ftruncate,open,pwrite64,rename");
 RUN("");
 ...
 OUTPUT("--------------wiredtiger_open");
 // lock == 3
 lock = open("./WiredTiger.lock", O_RDWR|O_CREAT|O_CLOEXEC, 0666);
-pwrite(lock, "WiredTiger lock file\n", 0x15, 0x0);
+pwrite64(lock, "WiredTiger lock file\n", 0x15, 0x0);
```

Comment:
@daveh86, I think the `pwrite64` changes are fine for Linux but may break running on OSX.  I don't know how much we care about that, but it works on OSX now.  (FTR, a way to resolve this may be to create a `PWRITE()` macro in the .run file, like how `FTRUNCATE` is defined now.  I'd vote for keeping things simple and not do that now.  That is, keep the pwrite64 changes you've made as is.)